src/textual/app.py

```diff
@@ -1451,6 +1437,41 @@ def save_screenshot(
             svg_file.write(screenshot_svg)
         return svg_path
 
+    def deliver_screenshot(
+        self,
+        filename: str | None = None,
+        path: str | None = None,
+        time_format: str | None = None,
+    ) -> str | None:
+        """Deliver a screenshot of the app.
+
+        This with save the screenshot when running locally, or serve it when the app
+        is running in a web browser.
+
+        Args:
+            filename: Filename of SVG screenshot, or None to auto-generate
+                a filename with the date and time.
+            path: Path to directory for output. Defaults to current working directory.
```

Comment:
It defaults to the user's download directory. Also might be worth noting that this doesn't apply on web.